src/components/common/DetailElement.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import Col from 'react-bootstrap/Col';
+
+const DetailElement = props => {
+  return (
+    <Col sm>
+      <strong>{props.header}</strong>

```

Comment:
I was considering that, but it would have to be something small, like h6. I think it makes more sense to keep it like this since it is not a section header technically.

Reply:
isn't it a section header? 

When it comes to size what really h1-h6 means is more of how important in this content this header is, what is it's hierarchy. Size can be always changed via css if needed

Reply:
Up to you man, this is just the title of each field, I do not think that should be an "h" element. The component is in common so I do not know in which hierarchy the element will be. If I make it h3 and there (hypothetically) are already h4 in the page where the component is used then it would not be correct.
Let me know what I should change it to.

Reply:
that's a good point, okay then, let's leave it in `strong` for now. We can always change that at some point if we figure out that, for example, it's being used only in places with h1 or something.